packages/datadog-plugin-rhea/src/index.js

```diff
@@ -118,30 +118,27 @@ function wrapDeliveryUpdate (update) {
 }
 
 function patchCircularBuffer (proto, instrumenter) {
-  Object.defineProperty(proto, 'session', {
+  Object.defineProperty(proto, 'outgoing', {
```

Comment:
Shouldn't this be done for both incoming and outgoing?

Reply:
It turns out it's not necessary since incoming and outgoing are always set in the constructor.

I'll add a safety net in case outgoing is no longer set.